src/tools/html-checker/main.rs

```diff
@@ -0,0 +1,65 @@
+use std::env;
+use std::path::Path;
+use std::process::{Command, Output};
+
+fn check_html_file(file: &Path) -> bool {
+    let mut to_mute = vec!["PROPRIETARY_ATTRIBUTE"];
+    if file.file_name().and_then(|f| f.to_str()) == Some("settings.html") {
+        // In `settings.html`, we have empty `<span>` for the toggles.
+        to_mute.push("TRIM_EMPTY_ELEMENT");
+    }
+    let to_mute_s = to_mute.join(",");
+    let mut command = Command::new("tidy");
+    command
+        .arg("-e")
+        .arg("-q")
```

Comment:
Can you use long options instead? I had to look up what these meant.
```suggestion
        .arg("-errors")
        .arg("-quiet")
```

Reply:
It becomes then `--error true` and `--quiet true` but sure.

Reply:
Ah nevermind, you can do `-quiet` or `--quiet true`. Dark magic.